src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Doctrine.Phpcr.Route.yml

```diff
@@ -0,0 +1,11 @@
+Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\Route:
```

Comment:
This is how jms serializer works. You need to write mapping for every extended class.